### PR TITLE
102224 about the conversion page

### PR DESCRIPTION
**Pull request by MarkRobinsonDfe**
Test Dev URL:-
https://localhost:44350/school/school-conversion-key-details?appId=2147483647&Urn=101934

### CODE REVIEW

**Review by wystanpalm-madetech — APPROVED** [review]
I would like this rebased properly so that the commit history is clean but I'm approving so it is up to you.

File: Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentHeadingViewModelTests.cs
Comment (CathLass) [inline Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentHeadingViewModelTests.cs:10]: Want to do these now?

File: Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentSectionViewModelTests.cs
Comment (CathLass) [inline Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentSectionViewModelTests.cs:10]: ditto

File: Dfe.Academies.External.Web/Pages/School/SchoolConversionKeyDetails.cshtml.cs
Comment (CathLass) [inline Dfe.Academies.External.Web/Pages/School/SchoolConversionKeyDetails.cshtml.cs:375]: Do you still need this commented-out stuff?

**Review by CathLass — COMMENTED** [review]
waiting to get a working url before I hit Approve

File: Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentSectionViewModelTests.cs
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentSectionViewModelTests.cs:10]: good spot, have added

File: Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentHeadingViewModelTests.cs
Comment (MarkRobinsonDfe) [reply Dfe.Academies.External.Web.UnitTest/ViewModels/SchoolConversionComponentHeadingViewModelTests.cs:10]: good spot, have added

**Comment by MarkRobinsonDfe** [comment]
> waiting to get a working url before I hit Approve

It works for me. Do you definitely have the secrets setup correctly with a valid trams API URI?

**Review by CathLass — APPROVED** [review]

**Pull request merged**
